tests/lib/rules/no-dupe-keys.js

```diff
@@ -77,8 +77,8 @@ ruleTester.run('no-dupe-keys', rule, {
 
   invalid: [
     {
-      filename: 'test.vue',
-      code: `
+      // filename: 'test.vue',
```

Comment:
Please create another tests for that, don't comment actual ones. Also try case with block comment too :)